app/live_display.py

```
@@ -0,0 +1,106 @@
+        self._running = False
+        if self._live:
+            self._live.stop()
+            self._live = None
+
+    def add_task(self, task_id: str, description: str, status: str = "pending"):
+        """Add a new task to track"""
+        with self._lock:
+            self._tasks[task_id] = {
+                "description": description,
+                "status": status,
+                "start_time": datetime.now(),
```

Ruff (0.14.7)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)